src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Character management", description = "Endpoints for managing characters")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/characters")
+public class CharacterController {
+    private static final int CHARACTER_COUNT = 826;
+    private final CharacterService characterService;
+
+    @GetMapping("/random-character")
```

Comment:
```suggestion
    @GetMapping("/random")
```
let's make naming more RESTful, resource oriented - /characters/random